CHANGELOG.md

```diff
@@ -17,9 +17,11 @@ Mapbox welcomes participation and contributions from everyone.
 * Fix Android memory leak when destroying platform view annotation manager. ([1568](https://github.com/mapbox/mapbox-maps-android/pull/1568))
 * Fix style getters for terrain, light and atmosphere resetting properties. ([1573](https://github.com/mapbox/mapbox-maps-android/pull/1573))
 * Fix possible ANR when destroying renderer. ([1567](https://github.com/mapbox/mapbox-maps-android/pull/1567))
+* Fix elevation of pole geometry when exaggerated terrain is used. ([1574](https://github.com/mapbox/mapbox-maps-android/pull/1574))
 
 ## Dependencies
 Bump gl-native to v10.8.0-beta.1, common to v23.0.0-beta.1. ([1568](https://github.com/mapbox/mapbox-maps-android/pull/1568))
+Bump gl-native to v10.8.0-beta.2. ([1574](https://github.com/mapbox/mapbox-maps-android/pull/1574))
```

Comment:
Guess we should actually replace entry above and note that 10.8.0-beta.2 is used

Reply:
Fixed.